python/pyspark/sql/functions.py

```diff
@@ -1675,15 +1675,18 @@ def array(*cols):
 @since(1.5)
 def array_contains(col, value):
     """
-    Collection function: returns True if the array contains the given value. The collection
-    elements and value must be of the same type.
+    Collection function: returns null if the array is null, true if the array contains the
+    given value, and false otherwise.
```

Comment:
I am willing to grep and replace too. Please let me know @holdenk.

FWIW, If deciding this drags this PR, I would like to say we should merge this first if you and @holdenk do not strongly feel about this.